client/post-editor/editor-tags/index.jsx

```diff
@@ -63,28 +63,27 @@ export default React.createClass( {
 	},
 
 	render: function() {
-		var tagNames = ( this.props.tags || [] ).map( function( tag ) {
+		const tagNames = ( this.props.tags || [] ).map( function( tag ) {
 			return tag.name;
 		} );
 
+		const helpText = isPage( this.props.post )
+			? this.translate( 'Use tags to associate more specific keywords with your pages.' )
+			: this.translate( 'Use tags to associate more specific keywords with your posts.' );
+
 		return (
-			<label className="editor-drawer__label">
-				<span className="editor-drawer__label-text">
+			<div>
```

Comment:
Correct - tags was the only place these labels were being used like a true `<label>` - I can change it back but figured since we are not doing so elsewhere - I used that as the standard instead.